contracts/abstract/JBProject.sol

```diff
@@ -75,16 +81,23 @@ abstract contract JBProject is Ownable {
     bool _preferClaimedTokens,
     address _token
   ) internal {
+    if (_projectId == 0) {
+      revert PROJECT_NOT_FOUND();
+    }
     require(_projectId != 0, '0x01: PROJECT_NOT_FOUND');
```

Comment:
is this being removed?

Reply:
@mejango, this is old code all require statements have been removed